src/Reflection/ReflectionClass.php

```diff
@@ -491,6 +495,60 @@ public function hasConstant(string $name) : bool
     }
 
     /**
+     * Get the reflection object of the specified class constant.
+     *
+     * Returns null if not specified.
+     *
+     * @param string $name
+     * @return ReflectionClassConstant|null
+     */
+    public function getReflectionConstant(string $name) : ?ReflectionClassConstant
+    {
+        $constants = $this->getReflectionConstants();
+
+        if (!array_key_exists($name, $constants)) {
+            return null;
+        }
+
+        return $constants[$name];
+    }
+
+    /**
+     * Returns array of constants for this class
+     *
+     * @param array $accumulator
+     * @param Node\Stmt $stmt
+     * @return array
+     */
+    private function processReflectionConstants(array $accumulator, Node\Stmt $stmt) : array
+    {
+        if ($stmt instanceof ConstNode) {
+            $const = ReflectionClassConstant::createFromNode($this->reflector, $stmt, $this);
```

Comment:
Oh, dang,  I just noticed that it is indeed used twice. Sorry, that's me misreading stuff :-\